releng/org.eclipse.xtext.dev-bom/pom.xml

```diff
@@ -0,0 +1,294 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.eclipse.xtext</groupId>
+	<artifactId>xtext-dev.bom</artifactId>
+	<version>2.17.0-SNAPSHOT</version>
+	<packaging>pom</packaging>
+	<description>Xtext Third Party Dependencies (Bill of Materials)</description>
+
+	<properties>
+		<!-- Versions -->
+		<asm-version>7.0</asm-version>
+		<eclipse-text-version>3.6.100</eclipse-text-version>
+		<core-resources-version>3.12.0</core-resources-version>
+		<core-runtime-version>3.13.0</core-runtime-version>
+		<emfCommon-version>2.12.0</emfCommon-version>
+		<emfEcore-version>2.12.0</emfEcore-version>
+		<emfChange-version>2.11.0</emfChange-version>
+		<emfCodegen-version>2.11.0</emfCodegen-version>
+		<emfMwe-version>1.3.21.201705291010</emfMwe-version>
+		<emfMwe2-version>2.9.1.201705291010</emfMwe2-version>
+		<equinoxApp-version>1.3.400</equinoxApp-version>
+		<equinoxCommon-version>3.9.0</equinoxCommon-version>
+		<equinoxRegistry-version>3.7.0</equinoxRegistry-version>
+		<eclipseCore-version>3.13.0</eclipseCore-version>
+		<eclipseOSGI-version>3.12.100</eclipseOSGI-version>
+		<guice-version>3.0</guice-version>
+		<icu4j-version>52.1</icu4j-version>
+		<jdtCore-version>3.13.102</jdtCore-version>
+		<jdtApt-version>1.3.110</jdtApt-version>
+		<jdtTool-version>1.2.101</jdtTool-version>
+		<junit-version>4.12</junit-version>
+		<junit5-version>5.1.0</junit5-version>
+		<log4j-version>1.2.17</log4j-version>
+		<!-- // When adjusting the `lsp4j` version here, do not forget to do the 
+			followings: // - Update the version in `IdeProjectDescriptor#pom`. // - Update 
+			the test expectations by running `CliWizardIntegrationTest#main`. // - Update 
+			the versions in the `/org.eclipse.xtext.ide/META-INF/MANIFEST.MF` and `/org.eclipse.xtext.ide.tests/META-INF/MANIFEST.MF` 
+			files. -->
+		<lsp4j-version>0.6.0</lsp4j-version>
+		<xpand-version>2.0.0</xpand-version>
+		<guava-version>[14.0,21.0]</guava-version>
```

Comment:
this change is intentional, correct?

Reply:
you mean managing guava version?

Reply:
i mean the new range

Reply:
for guava

Reply:
I would rather like to have a fixed version, but didn't you mention somewhere that it is intentional to have a range?

Reply:
see https://github.com/eclipse/xtext-lib/blob/3d3e03de9fec9c148576aaa2879999d13371144b/gradle/versions.gradle#L11